packages/swap/src/routes/v2/swap.ts

```diff
@@ -272,23 +280,21 @@ router.get(
       deposit: {
         amount:
           readField(swapRequest, failedSwap, 'depositAmount')?.toFixed() ?? pendingDeposit?.amount,
-        transactionRef: depositTransactionRef,
+        txRef: depositTransactionRef,
         txConfirmations: pendingDeposit?.transactionConfirmations,
         receivedAt: swapRequest?.depositReceivedAt?.valueOf(),
         receivedBlockIndex: swapRequest?.depositReceivedBlockIndex ?? undefined,
         ...(failedSwap && { failure: getDepositIgnoredFailedState(failedSwap) }),
       },
       swap: {
         ...rolledSwaps,
-        totalAmountSwapped: rolledSwaps?.totalAmountSwapped.toFixed(),
+        totalInputAmountSwapped: rolledSwaps?.totalInputAmountSwapped.toFixed(),
+        totalOutputAmountSwapped: rolledSwaps?.totalOutputAmountSwapped.toFixed(),
         lastExecutedChunk:
           rolledSwaps?.lastExecutedChunk && getSwapFields(rolledSwaps.lastExecutedChunk),
         currentChunk: rolledSwaps && getSwapFields(rolledSwaps.currentChunk),
-        ...swapEgressFields,
-        type: sortedSwaps?.[0].type,
         fees: aggregateFees,
-        srcChainRequiredBlockConfirmations,
-        estimatedDurationSeconds,
+        ...(showEgress && { egress: { ...swapEgressFields } }),
```

Comment:
i think i would move this out of the `swap` so its on the same level as `refund`

Reply:
then we would have `deposit`, `swap`, `egress`, `refund` on that level which feels right to me (?)